utility/DynamicLibrary.hpp

```diff
@@ -85,6 +85,11 @@ class DynamicLibrary : private utility::NonCopyable
     void *_handle;
 
     /**
+     * Path to the library
+     */
+    const std::string _path;
```

Comment:
If not, we can't refer to it later (e.g. in error messages). It is not a must, but it seemed useful on windows.